src/main/webapp/js/user-page-loader.js

```diff
@@ -105,12 +105,12 @@ function fetchAboutMe(){
   fetch(url).then((response) => {
     return response.text();
   }).then((aboutMe) => {
-    const aboutMeContainer = document.getElementById('about-me-container');
+    const aboutMeContainer = document.getElementById('about-me-container'); //finds tag with id about me container
```

Comment:
Please write comments with a space after `//`. Also, please capitalize comments.

`// Finds tag with id about me container`

Also, this comment is probably just noise -- it's good to remind yourself about why this code exists, but in this case it just repeats what the line is doing, so you should remove it.